packages/grid/backend/grid/core/config.py

```diff
@@ -15,6 +16,17 @@
 from pydantic import validator
 
 
+def str_to_int(bool_str: Any) -> int:
+    try:
+        return distutils.utils.strtobool(str(bool_str))
```

Comment:
![Screenshot 2023-11-21 at 9 54 07 AM](https://github.com/OpenMined/PySyft/assets/43314053/3814efb6-aea7-4fad-ae63-c890913fdbf0)
From the python documentation , it seems distutils is deprecated and planned for removal in python 3.12

https://docs.python.org/3.10/library/distutils.html#module-distutils

Maybe we could try another way?

Reply:
agree. reimplemented.